package.json

```diff
@@ -25,7 +25,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "yarn format"
+            "pre-commit": "yarn format && git add ."
```

Comment:
**code-style** does the `git add` for you: https://github.com/dhis2/code-style/blob/master/lib/git.js#L13